src/test/java/be/seeseemelk/mockbukkit/entity/PlayerMockTest.java

```diff
@@ -1128,4 +1128,14 @@ public void testFlightEventFired()
 		assertTrue(player.isFlying());
 		server.getPluginManager().assertEventFired(PlayerToggleFlightEvent.class);
 	}
+
+	@Test
+	public void testPlayerHide(){
+		PlayerMock player2 = server.addPlayer();
+		player.hidePlayer(null, player2);
+		assertFalse(player.canSee(player2));
+		player.showPlayer(null, player2);
```

Comment:
Plugin is marked as `NotNull`, this right here is a direct violation of that constraint and should result in a test failure.